docker-compose/_posts/2017-05-12-using-private-docker-images.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Using Private Docker Images
+short: Using Private Docker Images
+category: docker-compose
+order: 3
+development: true
+---
+
+Many teams use private Docker registries to host their images. To use your private images, refer to them in your Compose files or Dockerfiles and add your registry credentials to Runnable.
+
+---
+
+## Using Images in Your Files
+
+Use images in a Compose file by referring to them with the `image` configuration option:
+
+```yaml
+Services:
+  db:
+    image: example-registry.com:4000/postgresql
+```
+
+> **Tip:** Learn how to [use your Compose file]({{ site.baseurl }}/docker-compose/launching-your-first-environment) in Runnable.
+
+You can also use images in your services’ Dockerfiles:
```

Comment:
I think it's bit ambiguous if this is intentional. Did you intend to refer to multiple services, or just a single one? I would just drop the word.